generators/plugin/templates/dot_gitignore

```diff
@@ -0,0 +1,37 @@
+*.rbc
```

Comment:
You could just call this file gitignore I think most would know it becomes a .gitignore file